lib/action_account.rb

```diff
@@ -48,7 +48,26 @@ def subtask(name)
     }[name]
   end
 
-  class ReviewReject
+  class ReviewAction
+    def log_message(uuid:, log:, table:, messages:)
+      table << [uuid, log]
+      messages << '* `' + uuid + '` : `' + log + '`'
```

Comment:
I would handle the formatting at the outer layer in `ScriptBase` instead of here potentially?